src/olympia/blocklist/templates/blocklist/multiple_block.html

```diff
@@ -61,7 +61,9 @@ <h3>{% blocktrans with count=blocks|length %}{{ count }} Add-on GUIDs to Block{%
               ({{ block_obj.addon.average_daily_users }} {% trans 'users' %}).
               {{ block_obj.review_listed_link }}
               {{ block_obj.review_unlisted_link }}
-              {% if block_obj.id %}[<a href="{% url opts|admin_urlname:'change' block_obj.id %}">{% trans 'edit existing Block' %}</a>]{% endif %}
+              {% if block_obj.id %}
+                <span class="existing_block">[<a href="{% url 'admin:blocklist_block_change' block_obj.id %}">{% trans 'edit Block' %}</a>: {{ block_obj.min_version }} - {{ block_obj.max_version }}]</span>
```

Comment:
I am wondering why `edit` is not capitalized.

Reply:
No particular reason, it just wasn't capitalized in the previous iteration ("edit existing Block")

Reply:
https://github.com/mozilla/addons-server/pull/13113/commits/d09ec147dc145e2572743445b915711b4234f978